app/inbound/process-invoice-lines.js

```diff
@@ -0,0 +1,16 @@
+const db = require('../data')
+const enrichInvoiceLine = require('./enrichment/enrich-invoice-line')
+const validateInvoiceLine = require('./validate-invoice-line')
+
+const processInvoiceLines = async (invoiceLines, paymentRequestId, fundCode, transaction) => {
+  for (const invoiceLine of invoiceLines) {
+    // ignore any net lines
+    if (!invoiceLine.description.startsWith('N00')) {
```

Comment:
May be use a `const <nameofvariable>`

`if (!invoiceLine.description.startsWith(nameofvariable)) { `